BUILD

```diff
@@ -0,0 +1,11 @@
+package(default_visibility = ["//visibility:public"])
+
+java_library(
+    name = "grpc_java",
```

Comment:
I was using it just to reduce the number of deps in rules that use grpc-java. You can see an example here: https://github.com/simonhorlick/grpc-java-helloworld/blob/master/java/io/grpc/examples/helloworld/BUILD#L44